examples/layers/rendering/touch_input.dart

```diff
@@ -48,15 +48,15 @@ class RenderDots extends RenderBox {
   @override
   bool get sizedByParent => true;
 
-  /// By selecting the biggest value permitted by the incomming constraints
+  /// By selecting the biggest value permitted by the incoming constraints
   /// during layout, this function makes this render object as large as
   /// possible (i.e., fills the entire screen).
   @override
   void performResize() {
     size = constraints.biggest;
   }
 
-  /// Makes this render object hittable so that it receives pointer events.
```

Comment:
Maybe this was ok
https://www.merriam-webster.com/dictionary/hittable

Reply:
Reverted.  My dictionary let me down... :-(